cmd/spire-server/cli/run/run_test.go

```diff
@@ -680,6 +681,28 @@ func TestNewServerConfig(t *testing.T) {
 				require.Equal(t, 1337, c.Experimental.BundleEndpointAddress.Port)
 			},
 		},
+		{
+			msg: "bundle federates with section is parsed and configured correctly",
+			input: func(c *config) {
+				c.Server.Experimental.FederatesWith = map[string]federatesWithConfig{
+					"spiffe://example.org": {
+						BundleEndpointAddress:  "192.168.1.1",
+						BundleEndpointPort:     1337,
+						BundleEndpointSpiffeID: "spiffe://example.org/bundle/endpoint",
+						UseWebPKI:              true,
```

Comment:
Maybe we can also have a test to assert that the default value for UseWebPKI is false?

Reply:
Sure! Happy to add that.